<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainwindowClass</class>
 <widget class="QMainWindow" name="mainwindowClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>359</width>
    <height>358</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Cycle Dash for Windows</string>
  </property>
  <property name="windowIcon">
   <iconset resource="mainwindow.qrc">
    <normaloff>:/mainwindow/clipper_dashboard.png</normaloff>:/mainwindow/clipper_dashboard.png</iconset>
  </property>
  <property name="windowOpacity">
   <double>1.000000000000000</double>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>340</width>
      <height>299</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_11">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <widget class="QLabel" name="label">
             <property name="font">
              <font>
               <family>Fixedsys</family>
               <pointsize>20</pointsize>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>-.----</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_3">
             <property name="font">
              <font>
               <family>Fixedsys</family>
               <pointsize>20</pointsize>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>-.----</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_2">
             <property name="font">
              <font>
               <family>Fixedsys</family>
               <pointsize>20</pointsize>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>-.----</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_4">
             <property name="font">
              <font>
               <family>Fixedsys</family>
               <pointsize>20</pointsize>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>-.----</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_5">
             <property name="font">
              <font>
               <family>Fixedsys</family>
               <pointsize>20</pointsize>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>-.---- </string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QLabel" name="label_6">
             <property name="font">
              <font>
               <family>Fixedsys</family>
               <pointsize>20</pointsize>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>Ah</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_7">
             <property name="font">
              <font>
               <family>Fixedsys</family>
               <pointsize>20</pointsize>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>V</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_8">
             <property name="font">
              <font>
               <family>Fixedsys</family>
               <pointsize>20</pointsize>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>A</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_9">
             <property name="font">
              <font>
               <family>Fixedsys</family>
               <pointsize>20</pointsize>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>s</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_10">
             <property name="font">
              <font>
               <family>Fixedsys</family>
               <pointsize>20</pointsize>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>D</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_7">
           <item>
            <widget class="QLabel" name="label_21">
             <property name="text">
              <string>Max Voltage:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_22">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_8">
           <item>
            <widget class="QLabel" name="label_25">
             <property name="text">
              <string>Min Voltage:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_28">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="Line" name="line_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_9">
           <item>
            <widget class="QLabel" name="label_29">
             <property name="text">
              <string>Max Amps:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_30">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_10">
           <item>
            <widget class="QLabel" name="label_33">
             <property name="text">
              <string>Min Amps:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_34">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="Line" name="line">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_5">
           <item>
            <widget class="QLabel" name="label_13">
             <property name="text">
              <string>Max Speed:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_15">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_6">
           <item>
            <widget class="QLabel" name="label_14">
             <property name="text">
              <string>Min Speed:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_16">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QCheckBox" name="checkBox">
           <property name="text">
            <string>Save as .txt file</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="checkBox_2">
           <property name="text">
            <string>Include Timestamp</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="checkBox_3">
           <property name="text">
            <string>Include Clock Time</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QPushButton" name="pushButton_2">
           <property name="text">
            <string>Save As...</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_12">
         <item>
          <widget class="QLabel" name="label_11">
           <property name="text">
            <string>Seperate Elements With:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="comboBox_2">
           <item>
            <property name="text">
             <string>Tabs (Default)</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Commas ,</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Spaces</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Semicolons ;</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QPushButton" name="pushButton">
           <property name="text">
            <string>Connect</string>
           </property>
           <property name="autoDefault">
            <bool>true</bool>
           </property>
           <property name="default">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="comboBox">
           <item>
            <property name="text">
             <string>COM1</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>COM2</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>COM3</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>COM4</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>COM5</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>COM6</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>COM7</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>COM8</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>COM9</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>COM10</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>COM11</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>COM12</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>359</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionRefresh"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionRefresh">
   <property name="text">
    <string>Refresh</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="mainwindow.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>checkBox</sender>
   <signal>clicked()</signal>
   <receiver>mainwindowClass</receiver>
   <slot>checkbox_enables()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>33</x>
     <y>307</y>
    </hint>
    <hint type="destinationlabel">
     <x>57</x>
     <y>249</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>mainwindowClass</receiver>
   <slot>file_name()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>76</x>
     <y>335</y>
    </hint>
    <hint type="destinationlabel">
     <x>208</x>
     <y>259</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionRefresh</sender>
   <signal>triggered()</signal>
   <receiver>mainwindowClass</receiver>
   <slot>try_receive()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>182</x>
     <y>212</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>try_receive()</slot>
  <slot>get_time()</slot>
  <slot>checkbox_enables()</slot>
  <slot>file_name()</slot>
 </slots>
</ui>
